lib/eclipse/EclipseState/Schedule/TimeMap.cpp

```diff
@@ -151,6 +176,31 @@ namespace Opm {
         return monthNames;
     }
 
+    const std::map<std::string , int>& TimeMap::eclipseMonthIndices() {
+        static std::map<std::string , int> monthIndices;
+
+        if (monthIndices.size() == 0) {
+            monthIndices.insert( std::make_pair( "JAN" , 0 ));
```

Comment:
I would just shift these number with 1 - then they immediately fit in `TimeMap::mkdate( )`